python_modules/dagster/dagster_tests/core_tests/test_object_store.py

```diff
@@ -313,6 +318,30 @@ def test_file_system_object_store_with_composite_type_storage_plugin():
                 ['hello'], context, resolve_to_runtime_type(List_(String_)), ['obj_name']
             )
 
+    with yield_pipeline_execution_context(
```

Comment:
can you make an issue to make a generic wrapper around this

Reply:
#1231